.github/workflows/wasm32.yml

```
@@ -14,28 +14,22 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install wasmtime
-        run: curl https://wasmtime.dev/install.sh -sSf | sudo bash
+        run: |
+          apt-get update
+          apt-get install -y curl
+          curl https://wasmtime.dev/install.sh -sSf | bash
```

Can we pin this to a specific version?
```suggestion
          curl https://wasmtime.dev/install.sh -sSf | bash -s - --version v1.0.1
```